rxjava-support/src/main/java/tech/picnic/rx/RxSpring4Util.java

```diff
@@ -178,7 +177,7 @@ private RxSpring4Util() {}
   }
 
   private static <T, R extends Iterable<T>>
-      Function<? super Publisher<T>, DeferredResult<R>> publisherToDeferredResult(
+      Function<Publisher<T>, DeferredResult<R>> publisherToDeferredResult(
```

Comment:
Can we also drop the generic wildcard from the other `publisherToDeferredResult` overloads?

Reply:
Doh, not sure how I missed those. Will do! :)